### PR TITLE
docs: add M2 mac special install directions

### PR DESCRIPTION
**Pull request by allexiusw**
https://github.com/cypress-io/cypress-realworld-app/issues/1207 This documentation update is related to this issue that is already closed. 

I have tested the solution in M2 series and it works properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ TypeScript will be added as a local dependency to the project, so no need to ins
 yarn
 ```
 
-#### Mac M1 chip users will need to prepend `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true`.
+#### Mac M1, M2 chip users will need to prepend `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true`.
```
Comment (MikeMcC399) [inline README.md]: Maybe use "and" instead of ","?

Comment (mike-plummer) [reply README.md]: Could potentially future-proof it a bit by saying "Mac users with M-series chips..." since there will probably be an M3

Comment (allexiusw) [reply README.md]: Done... Thanks for the suggestion.

**Review by MikeMcC399 — APPROVED** [review]
Thanks for taking the initiative to help other users to benefit from your experience! 💪🏻

**Review by mike-plummer — APPROVED** [review]

**Pull request merged**